plugins/generator-1.18.2/forge-1.18.2/templates/item/item.java.ftl

```diff
@@ -253,5 +253,84 @@ public class ${name}Item extends Item {
 			stack.getCapability(CapabilityItemHandler.ITEM_HANDLER_CAPABILITY, null).ifPresent(capability -> ((ItemStackHandler) capability).deserializeNBT((CompoundTag) nbt.get("Inventory")));
 	}
 	</#if>
+
+	<#if data.shootConstantly>

```

Comment:
AFAIU, this part should only be generated if ranged actions are enabled.

Reply:
Don't start reviewing what I said was not finished. I literally tested nothing yet, except the UI.